static/app/gettingStartedDocs/node/serverlesscloud.tsx

```diff
@@ -0,0 +1,232 @@
+  projectId: string;
+  sentryInitContent: string;
+};
+
+const performanceIntegrations: string[] = [
+  `// enable HTTP calls tracing
+  new Sentry.Integrations.Http({ tracing: true }),`,
+  `// enable Express.js middleware tracing
+  new Sentry.Integrations.Express({ app }),`,
+  `// Automatically instrument Node.js libraries and frameworks
+  ...Sentry.autoDiscoverNodePerformanceMonitoringIntegrations(),`,
+];
+
+const performanceOtherConfig = `environment: params.INSTANCE_NAME,
+// Performance Monitoring
+// Capture 100% of the transactions, reduce in production!
+tracesSampleRate: 1.0, `;
+
+export const steps = ({
+  sentryInitContent,
+  ...props
+}: Partial<StepProps> = {}): LayoutProps['steps'] => [
+  {
+    type: StepType.INSTALL,
+    description: (
+      <p>{tct('Add [code:@sentry/node] as a dependency:', {code: <code />})}</p>
+    ),
+    configurations: [
+      {
+        language: 'bash',
+        code: `cloud install @sentry/node:`,
+      },
+    ],
+  },
+  {
+    type: StepType.CONFIGURE,
+    description: (
+      <p>
+        {tct('Sentry should be initialized as early in your app as possible.', {
+          code: <code />,
+        })}
+      </p>
+    ),
+    configurations: [
+      {
+        language: 'javascript',
+        code: `
+        import api from "@serverless/cloud";
+import * as Sentry from "@sentry/node";
+
+// or using CommonJS
+// const api = require("@serverless/cloud");
+// const Sentry = require('@sentry/node');
+
+Sentry.init({
+  ${sentryInitContent},
+});
+
+// RequestHandler creates a separate execution context, so that all
+// transactions/spans/breadcrumbs are isolated across requests
+api.use(Sentry.Handlers.requestHandler());
+// TracingHandler creates a trace for every incoming request
+api.use(Sentry.Handlers.tracingHandler());
+
+// All controllers should live here
+api.get("/", function rootHandler(req, res) {
+  res.end("Hello world!");
+});
+
+// The error handler must be before any other error middleware and after all controllers
+api.use(Sentry.Handlers.errorHandler());
+
+// Optional fallthrough error handler
+api.use(function onError(err, req, res, next) {
+  // The error id is attached to \`res.sentry\` to be returned
+  // and optionally displayed to the user for support.
+  res.statusCode = 500;
+  res.end(res.sentry + "\\n");
+});
+        `,
+      },
+      {
+        language: 'javascript',
+        description: (
+          <p>
+            {tct(
+              'The above configuration captures both error and performance data. To reduce the volume of performance data captured, change [code:tracesSampleRate] to a value between 0 and 1. You can verify the Sentry integration is working by creating a route that will throw an error:',
+              {code: <code />}
+            )}
+          </p>
+        ),
+        code: `
+        api.get("/debug-sentry", function mainHandler(req, res) {
+          throw new Error("My first Sentry error!");
+        });
+    `,
+      },
```

Comment:
I think we should just make this the "Verify" step. (and while were at it remove the text about lowering the sample rate - I think the comment in the init section should be enough)